modules/file-system/filestore/scripts/mount.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
This would probably benefit from having a set -e.
Also, could this not be an exact copy of modules/file-system/pre-existing-network-storage/scripts/mount.sh ?

Reply:
ah... It was a copy, but rebase didn't capture the copy. This is why we need that pre-commit check. I will make sure to update.

Reply:
This has been updated to now be an exact match of pre-existing-network-storage.